tests/BrowsershotTest.php

```diff
@@ -201,6 +187,7 @@ public function it_can_create_a_command_to_generate_a_screenshot()
                     'height' => 1080,
                 ],
                 'args' => [],
+                'type' => 'png',
```

Comment:
Why is 'png' added in every test? Can this be removed?

Reply:
It related to https://github.com/spatie/browsershot/pull/180

This change is not related with this PR. Should I remove it?
(I just tried to remove Travis Error)